src/join.t.sol

```diff
@@ -861,4 +932,27 @@ contract DssDeployTest is DssDeployTestBase {
         saiJoin.deny(address(this));
         saiJoin.join(address(this), 10);
     }
+
+    function testTokenManaged() public {
+        deployKeepAuth();
+        DSValue pip = new DSValue();
+
+        DSToken wbtc = new DSToken("WBTC");
```

Comment:
Shouldn't we better use the WBTC token to test the decimals conversion functionality as well?

Reply:
As this is the only not fail test we have for it.

Reply:
Agree.
Done.